dist/renderers/pixi/index.d.ts

```diff
@@ -19,6 +19,7 @@ export declare type TextIcon = {
 export declare type ImageIcon = {
     type: 'imageIcon';
```

Comment:
super annoying:  given that `image` isn't actually that descriptive, can we change this to `url: string` (all values should be valid urls)?  Also, might make sense to also expose an x and y property for offsets, defaulting to 0.